test/config/install_default.json

```diff
@@ -30,6 +30,27 @@
       "proxy": {
         "host": "",
         "port": ""
-      }
+      },
+      "os-install": [
+        {"workflow": "Graph.InstallESXi", "version": "6.0", "path": "/repo/esxi/6.0", "kvm": false},
+        {"workflow": "Graph.InstallCentOS", "version": "6.5", "path": "/repo/centos/6.5", "kvm": false},
+        {"workflow": "Graph.InstallUbuntu", "version": "trusty", "path": "/repo/ubuntu", "kvm": false},
+        {"workflow": "Graph.InstallESXi", "version": "5.5", "path": "/repo/esxi/5.5", "kvm": false},
```

Comment:
Is there any reason to not put the entries of the same OS together? Put same OS entries together will make it more clear to know the parameter difference for the same OS.